windows/src/engine/keyman32/appint/aiTIP.cpp

```diff
@@ -206,35 +207,56 @@ extern "C" __declspec(dllexport) BOOL WINAPI TIPProcessKey(WPARAM wParam, LPARAM
 	_td->state.msg.message = wm_keymankeydown;
 	_td->state.vkey = (WORD) wParam;
 
-	_td->state.lpkb = _td->lpActiveKeyboard->Keyboard;
+  if (isUsingCoreProcessor) {
+    // TODO: 5442 set the correct active keyboard - need with call backs
+    _td->state.lpCoreKb = _td->lpActiveKeyboard->coreKeyboard;
+
+  } else {
+    _td->state.lpkb       = _td->lpActiveKeyboard->Keyboard;
+    _td->state.startgroup = &_td->state.lpkb->dpGroupArray[_td->state.lpkb->StartGroup[BEGIN_UNICODE]];
+  }
 
   _td->state.windowunicode = TRUE;
-	_td->state.startgroup = &_td->state.lpkb->dpGroupArray[_td->state.lpkb->StartGroup[BEGIN_UNICODE]];
 
-	_td->state.NoMatches = TRUE;
-	_td->state.LoopTimes = 0;
-	_td->state.StopOutput = FALSE;
+  _td->state.NoMatches  = TRUE;
+  _td->state.LoopTimes  = 0;
+  _td->state.StopOutput = FALSE;
```

Comment:
All three of these are only used in `!isUsingCoreProcessor` so we could move them above